genetic_algorithms/problems/traveling_salesman_problem/visualization.py

```diff
@@ -0,0 +1,45 @@
+import pygame, sys
+from genetic_algorithms.algorithm_wrappers.algorithm_wrapper import AlgorithmWrapper
+from genetic_algorithms.problems.traveling_salesman_problem.problem_model import TravelingSalesmanModel
+from genetic_algorithms.problems.traveling_salesman_problem.state import TravelingSalesmanState
+
+class Visualization(AlgorithmWrapper):
+    def _perform_side_effects(self, model: TravelingSalesmanModel, state: TravelingSalesmanState):
+        pygame.init()
```

Comment:
Ja bym wyniósł to do `__init__`, ponieważ chyba nie potrzebujemy za każdym razem inicjalizować `pygame`?